### ISSUE TITLE
TypeError: can't access dead object

### ISSUE THREAD

**Issue by bkemper** [SparkPost/matchbox#317]
Error was from `getRectFor` specifically the `getBoundingClientRect` call.

```
function getRectFor(e) {
  var t = findDOMNode(e);
  return t ? t.getBoundingClientRect() : {};
}
```

See: https://sentry.io/organizations/sparkpost/issues/1454731741

**Comment by jonambas** [comment]
I believe we fixed this in 4.0.0, let me know if this shows up again

**Comment by bkemper** [comment]
I just checked Sentry and we haven't seen this error in the last 14 days!
